src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/TestUtil.java

```diff
@@ -195,4 +196,39 @@ public static PackageObjectFactory getPackageObjectFactory() throws CheckstyleEx
         return Optional.ofNullable(curNode);
     }
 
+    /**
+     * <p>
+     * Returns the JDK version as a number that is easy to compare.
+     * </p>
+     * <p>
+     * For JDK "1.8" it will be 8; for JDK "11" it will be 11.
+     * </p>
+     *
+     * @return JDK version as integer
+     */
+    public static int getJdkVersion() {
+        String version = System.getProperty("java.specification.version");
+        if (version.startsWith("1.")) {
+            version = version.substring(2);
+        }
+        return Integer.parseInt(version);
+    }
+
+    /**
+     * <p>
+     * Adjusts the expected number of flushes for tests that call {@link OutputStream#close} method.
+     * </p>
+     * <p>
+     * After <a href="https://bugs.openjdk.java.net/browse/JDK-8220477">JDK-8220477</a>
+     * there is one additional flush from {@code sun.nio.cs.StreamEncoder#implClose}.
+     * </p>
+     */
+    public static int adjustFlushCountForOutputStreamClose(int expectedFlushCount) {
+        int flushCount = expectedFlushCount;
```

Comment:
```
    public static int adjustFlushCountForOutputStreamClose(int flushCount) {
        int result = flushCount;
```

Reply:
done

Reply:
not done.

please do `(int expectedFlushCount)` --> `(int flushCount)`

util is stateless and context less environment, it should not know what is `expected`, it just do conversion from one value to another.

Reply:
ok, done.